web_frontend/cloudscheduler/glintwebui/glint_views.py

```diff
@@ -832,6 +867,16 @@ def upload(request, group_name=None):
             config.db_close()
             return render(request, 'glintwebui/upload_image.html', context)
 
+        # added code in if to reorganize or compress then convert the RAW/QCOW2 file-------------------------
+        virt_sparsify = bool(request.POST.get('operation1'))
```

Comment:
as mentioned in another file we should probably change the name of these boxes to be something relevant.